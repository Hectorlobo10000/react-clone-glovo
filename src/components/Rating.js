/**@jsx jsx */
import React from "react";
import { jsx } from "@emotion/core";
import { Icon } from "semantic-ui-react";

function Rating({ rating = 5 }) {
  const container = { display: "flex" };

  const p = {
    margin: "0px 10px 0px 0px",
    fontSize: "27px"
  };

  return (
    <div css={container}>
      <p css={p}>{rating}</p>
      <Icon name="star outline" size="large" />
    </div>
  );
}

export default Rating;
