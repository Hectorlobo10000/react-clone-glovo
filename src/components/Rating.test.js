import React from "react";
import { render } from "@testing-library/react";

import Rating from "./Rating";

test("Login Form on Submit", () => {
  const { asFragment } = render(<Rating ratingNumber={5} />);
  expect(asFragment()).toMatchSnapshot();
});
