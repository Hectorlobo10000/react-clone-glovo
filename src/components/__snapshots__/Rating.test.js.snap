// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Form on Submit 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: flex-end;
  -ms-flex-direction: flex-end;
  flex-direction: flex-end;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
    class="emotion-0"
  >
    <p />
    <div>
      <i
        aria-hidden="true"
        class="star outline small icon"
      />
    </div>
  </div>
</DocumentFragment>
`;
